third_party/arcs/examples/BUILD

```diff
@@ -0,0 +1,11 @@
+load("//build_defs:arcs.bzl", "arcs_manifest_proto")
+
+arcs_manifest_proto(
```

Comment:
It looks like we have this example Arcs manifest file tree and a recipe to build it, but I don't think it's hooked up to `@raksha//:all_tests` anywhere. As such, the CI isn't doing anything to check the health of the parser. Could you add a test that does that, even if it's a simple smoke test that just parses these manifests, checks that a proto file is produced, and checks that the exit code of the parser is 0?

Reply:
Yep, will do. Could you point me to how you set up the CI for the facts tests?

Reply:
Sure. Github Actions uses the source file `.github/workflows/main.yml` to figure out what to do. As you can see from that config file, it has a single workflow named `CI` which, upon any `push` or `pull_request` to the `main` branch, runs the single job `test` in `jobs`. `test` just checks out the repo, uses `apt-get` to grab a bunch of Ubuntu packages that Souffle needs, and runs `bazel test :all_tests` on the checked-out repo.

If you just edit the build so that `bazel test :all_tests` eventually kicks off a test of the `manifest2proto` parser, that should do it. If it needs any additional Ubuntu packages, you can add that to the `apt-get` invocation in that `main.yml` file.

If you want to do something more than the above and need to further edit the `main.yml` file, editing it within the GitHub web UI provides some helpful hints from GitHub Marketplace about little code snippets you can use to cobble some actions together.

Reply:
I ended up adding another step in the .yml file.

Reply:
Works for me.